downloader/settings.py

```diff
@@ -11,23 +11,24 @@
 """
 
 import os
+from decouple import config
+from decouple import config, Csv
```

Comment:
why have you duplicated this line?

Reply:
Actually at first I was trying with Allowed hosts .Sorry I m correcting it